ocs_ci/helpers/dr_helpers_ui.py

```diff
@@ -392,3 +403,139 @@ def verify_failover_relocate_status_ui(
         log.info("Close button found")
         acm_obj.do_click_by_xpath("//*[text()='Close']")
         log.info("Data policy modal page closed")
+
+
+def verify_mco_console_plugin():
```

Comment:
can't find where we use this new function. Did we add this validation to test or somewhere?

Reply:
> can't find where we use this new function. Did we add this validation to test or somewhere?

good point, added it now